trl/trainer/rloo_trainer.py

```diff
@@ -527,7 +522,7 @@ def __init__(
                     max_num_seqs=self.args.per_device_train_batch_size
                     * self.vllm_tensor_parallel_size
                     * self.args.steps_per_generation,
-                    max_model_len=max_model_len,
+                    # max_model_len=self.args.vllm_max_model_length,  # TODO
```

Comment:
You want to add

https://github.com/huggingface/trl/blob/20cc2e1d364684eb1bad5dd305798bc1108da4fb/trl/trainer/grpo_config.py#L150

in this PR or in a next one?